### ISSUE TITLE
Source Order and Skipped Heading Levels for My Files, Collections Listing

### ISSUE THREAD

**Issue by mtribone** [projecthydra/sufia#974]
Headings provide document structure and facilitate keyboard navigation by users of assistive technology.  Our views for handling the listing of files, collections, highlighted, and shares need to be updated so that the h1 and subsequent headings come first in the html source order. Also we skip some heading levels, going from h1 to h4, and they need to be ordered properly with the headings on the facets. Check with Blacklight's updates for source order and heading level.

https://github.com/projectblacklight/blacklight/pull/1124

Swap columns in the view so that the listing comes first in the source order and use Bootstrap's "pull-right" class to handle the output visually.

Either use HTML5 and create sections or asides to accommodate multiple h1 elements and heading levels or use HTML 4.0 and only use one h1 and properly ordered heading levels such as h1, h2, h3, etc.

**Comment by mejackreed** [comment]
You probably saw this, but just a heads up some semantic classes added in https://github.com/projectblacklight/blacklight/pull/1148 to help ease this transition.

**Comment by mtribone** [comment]
@mejackreed Thanks! I did see this, but thanks for pointing it out. I'll probably tackle this after Sufia 6 comes out and we test how well it plays with ScholarSphere. Might be too much before our code freeze at the end of the week.
